i2c/max/max518x_test.go

```diff
@@ -100,6 +107,67 @@ func TestMAX581xSetVoltage(t *testing.T) {
 	}
 }
 
+func TestMAX581xConn(t *testing.T) {
+	c, _ := i2c.Open(iotest.NewI2CDriver(iotest.NewI2CConn()), 0x1)
+	dac, _ := NewMAX5813(c, 2.048)
+	assert.Equal(t, c, dac.Conn())
+}
+
+// TestMAX581xSetInputCodeWithInvalidChannel calls dac.SetInputCode with a
+// input code that lies outside the range of the DAC.
+func TestMAX581xSetInputCodeWithInvalidCode(t *testing.T) {
+	var tests = []struct {
+		dac  dac.DAC
+		code int
+	}{
+		{MAX5813{}, -1},
+		{MAX5813{}, 256},
+		{MAX5814{}, -1},
+		{MAX5814{}, 1024},
+		{MAX5815{}, -1},
+		{MAX5815{}, 4096},
+	}
+
+	for _, test := range tests {
+		assert.NotNil(t, test.dac.SetInputCode(test.code, 1))
```

Comment:
Just checking if the error is not nil does not guarantee the error thrown is the one expected, maybe something else broke and returned an error. I think it might be better to compare the error to an expected error.